src/main/scala/viper/gobra/ast/frontend/PrettyPrinter.scala

```diff
@@ -47,7 +47,9 @@ class DefaultPrettyPrinter extends PrettyPrinter with kiama.output.PrettyPrinter
     case n: PStructClause => showStructClause(n)
     case n: PInterfaceClause => showInterfaceClause(n)
     case n: PBodyParameterInfo => showBodyParameterInfo(n)
+    case n: PConditionalMeasureCollection => showConditionalMeasureCollection(n)
```

Comment:
Here, I do not understand why you just match on `PConditionalMeasureCollection` instead of matching a `PTerminationMeasure`